dnf-plugins-core.spec

```diff
@@ -1,4 +1,4 @@
-%{?!dnf_lowest_compatible: %global dnf_lowest_compatible 2.7.0}
+%{?!dnf_lowest_compatible: %global dnf_lowest_compatible 2.8.0}
```

Comment:
@edynox Please could you change to 2.8.1

Reply:
@j-mracek done